enhancements/kube-apiserver/audit-policy.md

```diff
@@ -230,7 +252,10 @@ apiVersion: config.openshift.io/v1
 spec:
   ...
   audit:
-    profile: Default | WriteRequestBodies | AllRequestBodies
+    profile: None | Default | WriteRequestBodies | AllRequestBodies
```

Comment:
Do we want audit.profile listed? Is customRules.group.profile on line 258 enough? Or are lines 255 and 258 both needed?
Is None a choice we want for profiles? I don't know if that's an option right now.

Reply:
`None` is new. We need it to reduce logging, e.g. for the case where logs are restricted to authenticated oauth users.

Reply:
`None` is really helpful for users who want to reduce the amount of audit logs.
But the current limitation `With None set for (2), the cluster is only under limited support, i.e. for support cases, the customer will be asked to set this to one of the other three values. We will document this in the API documentation.` prevents to use this profile. There are no users who don't want Red Hat support. So I think no users use this profile even if it's implemented. Are there any expected use cases for `None` profile? 
I understand using `None` only is under limited support. But if customers use `None` + some customRules, is it not the limited support?

Reply:
You can upgrade the cluster and we don't say no to support requests. But if we require audit logs, the user is expected to switch the profile to Default or higher. Am happy to hear a better wording for this.

Basically, the user story this should prevent: "No, we cannot increase audit logging for this production cluster because of X. So we cannot provide audit logs to support".

cc @sferich888 @Anandnatraj

Reply:
Thank you for explanation. I misread the above sentence. I agree to this policy. For support requests, it's natural to request further information to troubleshoot customer's issues if required.